sites/all/modules/unl_cas/unl_cas.module

```
@@ -125,6 +125,9 @@ function unl_cas_validate_ticket() {
   if ($auth) {
     $username = $cas->getUsername();
     $user = unl_cas_import_user($username);
+    if (!$user) {
+      drupal_set_message('An error occured importing the user.', 'error');
```

should we return here too? fail early?